package-checks.cake

```diff
@@ -0,0 +1,237 @@
+    string[] CONSOLE_FILES = { "nunit3-console.exe", "nunit3-console.exe.config" };
+
+    bool isOK =
+        CheckNuGetPackage(
+            "NUnit.Console",
+            HasFile("LICENSE.txt")) &
+        CheckNuGetPackage(
+            "NUnit.ConsoleRunner",
+            HasFiles("LICENSE.txt", "NOTICES.txt", "CHANGES.txt"),
+            HasDirectory("tools").WithFiles(CONSOLE_FILES).AndFiles(ENGINE_FILES),
+            HasDirectory("tools/agents/net20").WithFiles(AGENT_FILES),
+            HasDirectory("tools/agents/net40").WithFiles(AGENT_FILES)) &
+        CheckNuGetPackage("NUnit.Engine",
+            HasFiles("LICENSE.txt", "NOTICES.txt", "CHANGES.txt"),
+            HasDirectory("lib/net20").WithFiles(ENGINE_FILES),
+            HasDirectory("lib/netstandard1.6").WithFiles(ENGINE_FILES),
+            HasDirectory("lib/netstandard2.0").WithFiles(ENGINE_FILES),
+            HasDirectory("contentFiles/any/agents/net20").WithFiles(AGENT_FILES),
+            HasDirectory("contentFiles/any/agents/net40").WithFiles(AGENT_FILES)) &
+        CheckNuGetPackage(
+            "NUnit.Engine.Api",
+            HasFile("LICENSE.txt"),
+            HasDirectory("lib/net20").WithFile("nunit.engine.api.dll"),
+            HasDirectory("lib/netstandard1.6").WithFile("nunit.engine.api.dll"),
+            HasDirectory("lib/netstandard2.0").WithFile("nunit.engine.api.dll")) &
+        CheckNuGetPackage(
+            "NUnit.Runners",
+            HasFile("LICENSE.txt")) &
+        CheckChocolateyPackage(
+            "nunit-console-runner",
+            HasDirectory("tools").WithFiles("LICENSE.txt", "NOTICES.txt", "CHANGES.txt", "VERIFICATION.txt").AndFiles(CONSOLE_FILES).AndFiles(ENGINE_FILES),
+            HasDirectory("tools/agents/net20").WithFiles(AGENT_FILES),
+            HasDirectory("tools/agents/net40").WithFiles(AGENT_FILES)) &
+        CheckChocolateyPackage(
+            "nunit-console-with-extensions",
+            HasDirectory("tools").WithFiles("LICENSE.txt", "NOTICES.txt", "VERIFICATION.txt")) &
+        CheckZipPackage(
+            "NUnit.Console",
+            HasFiles("LICENSE.txt", "license.rtf", "NOTICES.txt", "CHANGES.txt"),
+            HasDirectory("bin/net20").WithFiles("nunit3-console.exe", "nunit3-console.exe.config").AndFiles(ENGINE_FILES),
+            HasDirectory("bin/net35").WithFiles("nunit3-console.exe", "nunit3-console.exe.config").AndFiles(ENGINE_FILES),
+            HasDirectory("bin/netstandard1.6").WithFiles(ENGINE_FILES),
+            HasDirectory("bin/netstandard2.0").WithFiles(ENGINE_FILES),
+            HasDirectory("bin/netcoreapp1.1").WithFiles(ENGINE_FILES),
+            HasDirectory("bin/netcoreapp1.1").WithFiles(ENGINE_FILES),
+            HasDirectory("bin/agents/net20").WithFiles(AGENT_FILES),
+            HasDirectory("bin/agents/net40").WithFiles(AGENT_FILES)) &
+        CheckMsiPackage("NUnit.Console", HasDirectory("NUnit.org/nunit-console")); // Placeholder for later implementation
```

Comment:
This is a great step in the right direction, thanks for adding these! Can I suggest adding on the addins files? Given they're about to get a little more complicated, with different files for different levels of different packages, this would be a great way to validate that they're being gathered up correctly. 😄